projects/plugins/search/readme.txt

```diff
@@ -4,7 +4,7 @@ Tags: jetpack, search
 Requires at least: 5.7
 Requires PHP: 5.6
 Tested up to: 5.8
-Stable tag: 1.0
+Stable tag: 0.1.0
```

Comment:
Opened https://github.com/Automattic/jetpack/pull/20818 to handle this in future generated plugins.